app/src/pages/inside/projectSettingsPageContainer/content/testLibrary/testData/testData.scss

```diff
@@ -14,5 +14,11 @@
  * limitations under the License.
  */
 
-.test-data {
+.datasets-wrapper {
+  display: flex;
+  gap: 24px;
+}
+
+.how-to-get-started {
```

Comment:
Please nest classes to have one named wrapper to decrease changes to mess with global styles